### DISCUSSION TITLE
企业微信Webhook回调可用

### DISCUSSION THREAD

**Discussion by KoWming** [jeessy2/ddns-go#324] [Ideas]
下载企业微信→左上角三横杠→全新创建企业→个人组件团队(创建个人的企业群聊)，
进入群聊添加 [群机器人] 复制机器人Webhook地址填入ddns-go后台Webhook URL地址栏，
在RequestBody栏填入回调函数，
格式：

{ "msgtype": "markdown","markdown": { "content": "公网IP变更：\n 新IPV6地址：#{ipv6Addr} \n 已解析的域名：#{ipv6Domains} \n 域名更新结果：#{ipv6Result}", } }

模拟测试Webhook，返回 Webhook调用成功, 返回数据: {"errcode":0,"errmsg":"ok"}  字样即成功，企业微信个人群机器人会推送回调信息。

**Comment by KoWming** [comment]
手机上即可操作，不需要电脑进行设置，比较方便。

**Comment by jeessy2** [comment]
有啥新的Webhook通知。可提交到https://github.com/jeessy2/ddns-go/issues/327

**Reply by KoWming** [reply]
好的，感谢。
